wiki/Community/Project_Loved/en.md

```diff
@@ -15,13 +15,13 @@ To submit a beatmap for Loved, please visit the [beatmap submission page](https:
 - [osu!catch submissions](https://loved.sh/submissions/fruits)
 - [osu!mania submissions](https://loved.sh/submissions/mania)
 
-Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps reaching 80% "Yes" votes at their polls' end time will be moved to the Loved category.
+Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps reaching 80% "Yes" votes at their polls' end time will be moved to the Loved category. If a beatmap fails the voting they are eligible for a second shot at Loved.
```

Comment:
seems misleading as it implies that a failed beatmap is guaranteed a second shot, which is not the case.

something like this could work better:
```suggestion
Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps reaching 80% "Yes" votes at their polls' end time will be moved to the Loved category. A failed vote does not mean a beatmap would lose its chance to be re-nominated in the future.
```

Reply:
But doesn't your suggestion leave the fact out that it can only have a second chance. Yours sounds like it can fail voting without a limit of attempts